recipes/libglvnd/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "libglvnd" %}
+{% set version = "1.7.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://gitlab.freedesktop.org/glvnd/libglvnd/-/archive/v{{ version }}/libglvnd-v{{ version }}.tar.bz2
+    sha256: d0e1925a3c9aee0143b8c181ac31d5637c8faa081759c277b8e16c7075612c11
+    patches:
+      - libglvnd_look_for_egl_icd_in_system_location.patch
+
+build:
+  skip: true  # [not linux]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - meson
+    - pkg-config
+    - ninja
+  host:
+    - xorg-libx11
+    - xorg-libxext
+    - xorg-xextproto
+    - xorg-glproto
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libEGL.so.1
+    - test -f $PREFIX/lib/libGL.so.1
+    - test -f $PREFIX/lib/libGLESv1_CM.so.1
+    - test -f $PREFIX/lib/libGLESv2.so.2
+    - test -f $PREFIX/lib/libGLX.so.0
+    - test -f $PREFIX/lib/libOpenGL.so.0
+    - test -f $PREFIX/lib/libGLdispatch.so.0
```

Comment:
I'd prefer these to be in `libglvnd` and the rest to be in `libgl-devel`. I can make that change if you prefer

Reply:
Please go ahead with that. I gave you access to the repository.

Reply:
For the headers, `xorg-glproto` (mandatory headers in conjunction) must also be a runtime dependency.